src/evict/evict_lru.c

```diff
@@ -374,7 +408,8 @@ __evict_thread_stop(WT_SESSION_IMPL *session, WT_THREAD *thread)
      * The only time the first eviction thread is stopped is on shutdown: in case any trees are
      * still open, clear all walks now so that they can be closed.
      */
-    WT_WITH_PASS_LOCK(session, ret = __evict_clear_all_walks(session));
+    WT_WITH_PASS_LOCK(session, __clear_saved_walk_tree(session);
```

Comment:
Why do we save tree walk and then clear page walk?

Reply:
This is an example of automatic formatter's failure. These two lines are all within the PASS_LOCK.